app/controllers/session_controller.rb

```diff
@@ -897,4 +904,23 @@ def domain_redirect_allowed?(hostname)
 
     allowed_domains.split("|").include?(hostname)
   end
+
+  def enqueue_password_reset_for_user(user)
+    RateLimiter.new(
```

Comment:
I got myself into trouble with this. I wanted to test the rate limiter, so I did this 6 times and of course got limited, fine so far. I set a new password with one of the reset password emails. But then, I cannot reset my password again that day :sweat_smile: I wonder if we should reset this _after_ the user completes the whole reset password flow? Or at least minus one from it? IDK maybe it's not a real problem in practice.

Reply:
I'm not against resetting the rate limit but I must admit I'm not sure if it is a problem common enough for us to build in a reset. I do think we can do this in another PR though so as to keep the change being introduced here smaller.

Reply:
Yep no problem, in reality it's probably not a huge deal.